packages/upload-media/src/store/actions.ts

```diff
@@ -59,6 +62,23 @@ type ThunkArgs = {
 	registry: WPDataRegistry;
 };
 
+/**
+ * Returns an action object that pauses all processing in the queue.
```

Comment:
Looks like I didn't use the correct docblock here in the media-experiments plugin.